src/fides/api/schemas/redis_cache.py

```diff
@@ -95,7 +97,7 @@ def labeled_dict(
         self, include_default_labels: Optional[bool] = False
     ) -> Dict[str, Any]:
         """Returns a dictionary that preserves the labels for all custom/labeled identities."""
-        d = super().dict()
+        d = super().model_dump()
```

Comment:
I think this one was intentional because `model_dump()` is overridden here too?

Reply:
I need to probably figure out how to do this better

Reply:
Or maybe it works because I later changed it to loop through `for field in self.__pydantic_extra__ or {}` in this function body